### PR TITLE
New logic for moving between lists

### PR DESCRIPTION
**Pull request by alexreardon**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-beautiful-dnd",
-  "version": "12.1.1",
+  "version": "12.2.0-alpha.1",
```
Comment (danieldelcore) [inline package.json]: Might want to make sure we remove this

File: test/unit/state/get-droppable-over/preferencing.spec.js [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+  descriptor: {
+    id: 'my draggable',
+    index: 0,
+    type: droppableLarge.descriptor.type,
+    droppableId: droppableLarge.descriptor.id,
+  },
+  borderBox: droppableLarge.client.borderBox,
+});
+
+it('should prefer lists that are further away in the case that multiple lists are hit', () => {
```
Comment (alexreardon) [inline test/unit/state/get-droppable-over/preferencing.spec.js]: @danieldelcore can you please add a comment explaining which two cases are being hit?

File: package.json (hunk 1 above)
Comment (alexreardon) [reply package.json]: It's alright - it is going to `dev`

Comment (alexreardon) [reply package.json]: and it is fine to go into `master` with that. Master != production right now

**Pull request merged**